lnst/RecipeCommon/Perf/Measurements/Results/NeperFlowMeasurementResults.py

```diff
@@ -27,7 +27,7 @@ def warmdown_start(self):
 
         return min(
             [
-                parallel[-self.warmup_duration].start_timestamp
+                parallel.start_timestamp - self.warmup_duration
```

Comment:
Shouldn't this be `parallel.end_timestamp - self.warmup_duration` ?

Reply:
fixed.